npm_and_yarn/lib/dependabot/npm_and_yarn/file_fetcher.rb

```diff
@@ -9,6 +10,7 @@
 
 module Dependabot
   module NpmAndYarn
+    # rubocop:disable Metrics/ClassLength
```

Comment:
I'll follow this up with PRs for Yarn so I'll break out a new class to get rid of this.